tests/bluechi_machine_lib/bluechictl.py

```diff
@@ -119,23 +119,47 @@ def stop(self):
 
 
 class BluechiCtl:
```

Comment:
I am not sure we should keep two implementations of `BluechiCtl`. I think moving the `bluechi_test.bluechictl` to `bluechi_machine_lib` and extending it with the functions of the current `bluechi_machine_lib` would be best. The `subprocess.Popen` can be encapsulated in another `Client` implementation, e.g. `ProcessClient`.

Not in this PR, but would be great to do this in a follow-up.

Reply:
But these are two different things.
One is running locally. The other runs with `exec`

Reply:
`exec_run` is defined on the `Client` interface. So having a Client-subclass that implements it using the `subprocess.popen` (or uses the `Command` class for it) encapsulates that.

Reply:
But this way you will copy it to `bluechi_machine_lib`, and then you are getting code that you can't run on the machine - on the machine

Reply:
But as I wrote: maybe as a follow-up :)